components/form/form-directive.ts

```diff
@@ -49,7 +50,7 @@ class FormController {
 
             if (uiSchemaMaybe === undefined || uiSchemaMaybe === null || uiSchemaMaybe === "") {
                 // resolve JSON schema, then generate ui Schema
-                JsonRefs.resolveRefs(schema, {}, (err, resolvedSchema, meta) => {
```

Comment:
The meta resolution data varaible of json-refs. It was unused and caused a warning, hence I decided to remove it.